# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apache/openssl-0.9.8.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./modules/ssl/ssl_engine_pphrase.c.orig	2005-07-14 23:32:39.040813840 -0400
+++ ./modules/ssl/ssl_engine_pphrase.c	2005-07-14 23:35:34.285860200 -0400
@@ -30,6 +30,13 @@
                                            -- Clifford Stoll     */
 #include "mod_ssl.h"
 
+#ifndef PEM_F_DEF_CALLBACK
+# ifdef PEM_F_PEM_DEF_CALLBACK
+/* In OpenSSL 0.9.8 PEM_F_DEF_CALLBACK was renamed */
+#  define PEM_F_DEF_CALLBACK PEM_F_PEM_DEF_CALLBACK
+# endif
+#endif
+
 /*
  * Return true if the named file exists and is readable
  */
